spec/configuration_spec.rb

```diff
@@ -28,13 +28,8 @@ class Validator::ConfigTestValidator; end
       it "resolves a check named ConfigTestValidator" do
         file = double(:io, read: "---\npackage_checks:\n - ConfigTestValidator\n")
         config = described_class.new(file)
-        expect(config.package_checks).to eql([Validator::ConfigTestValidator])
-      end
-
-      it "resolves a check named ConfigTestValidator" do
-        file = double(:io, read: "---\npackage_checks:\n - Validator::ConfigTestValidator\n")
```

Comment:
accepting both `ConfigTestValidator` and `Validator::ConfigTestValidator` needlessly complicates things, no reason to have the second form, so I removed it.